src/App/App.js

```diff
@@ -14,8 +14,8 @@ import fetcher from '../API/APIcalls';
 import './App.scss';
 
 export class App extends Component{
-  constructor() {
-    super()
+  constructor(props) {
+    super(props)
 
```

Comment:
we need to remove this props since there is nothing being pass from the parent `index.js`